proto/viam/app/v1/app.proto

```diff
@@ -124,6 +124,9 @@ service AppService {
   // Gets the Robot API Keys for the robot
   rpc GetRobotAPIKeys(GetRobotAPIKeysRequest) returns (GetRobotAPIKeysResponse);
 
+  // Rename an API Key
+  rpc RenameKey(RenameKeyRequest) returns (RenameKeyResponse);
```

Comment:
Nit: move this definition next to the rest of the *Key rpc method definitions around line 210.